src/main/java/apoc/meta/Meta.java

```diff
@@ -40,28 +41,52 @@
     public KernelTransaction kernelTx;
 
     public enum Types {
-        INTEGER,FLOAT,STRING,BOOLEAN,RELATIONSHIP,NODE,PATH,NULL,UNKNOWN,MAP,LIST;
+        INTEGER,FLOAT,STRING,BOOLEAN,RELATIONSHIP,NODE,PATH,NULL,UNKNOWN,MAP,LIST,POINT,DATE,DATETIME,LOCALTIME,LOCALDATETIME,TIME,DURATION;
+
+        private static Map<Class<?>, Class<?>> numberMapping = new HashMap(){{
+            put(double.class, Double.class);
+            put(float.class, Float.class);
+            put(int.class, Integer.class);
+            put(long.class, Long.class);
+            put(short.class, Short.class);
+        }};
+
+        @Override
+        public String toString() {
+            switch (this){
+                case LIST:
+                    return "LIST OF ANY?";
+                default:
+                    return super.toString();
```

Comment:
Vedendo sulla console di Neo4j con CALL dbms.procedures() mi esce ad esempio `STRING?`
```suggestion
                    return super.toString() + "?";
```